dev-libs/aws-sdk-cpp/aws-sdk-cpp-1.7.178.ebuild

```diff
@@ -0,0 +1,147 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit cmake-utils python-single-r1
+
+DESCRIPTION="AWS SDK for C++"
+HOMEPAGE="https://aws.amazon.com/sdk-for-cpp/"
+SRC_URI="https://github.com/aws/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+MODULES=(
+	access-management apigateway backup batch budgets chime cloud-desktop cloud-dev cloud-media cloud-mobile
+	cloudformation cloudfront cloudhsm cloudsearch cloudwatch cognito config dlm ec2 ecr ecs eks elasticache
+	elasticbeanstalk elasticloadbalancing elasticmapreduce email es events fsx globalaccelerator iot kinesis kms
+	lambda lex license-manager lightsail machinelearning macie managedblockchain marketplace opsworks organizations
+	other personalize polly queues rds rekognition resource-groups route53 s3 sagemaker secretsmanager securityhub
+	serverlessrepo shield sns sqs textract transcribe translate waf
+)
+
+IUSE="+http +rtti +ssl static-libs test unity-build ${MODULES[*]}"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="
+	http?	( net-misc/curl:= )
```

Comment:
I know it looks better, but let's follow the default style and keep one space here between `? (` :)

Reply:
*meh* I used one tab. Indentation is tabbed any way.
but ... *sigh ... Check!